changelog/unreleased/bump-reva.md

```diff
@@ -2,4 +2,28 @@ Enhancement: Update reva
 
 Update reva to latest edge
 
+*   Bugfix [cs3org/reva#3963](https://github.com/cs3org/reva/pull/3963): Treesize interger overflows
```

Comment:
```suggestion
*   Bugfix [cs3org/reva#3963](https://github.com/cs3org/reva/pull/3963): Treesize integer overflows
```

Reply:
It is wrong in reva, so would need to be corrected there.